forest/bonsai/tests/test_simulate_gps_data.py

```diff
@@ -1,8 +1,14 @@
 """Tests for simulate_gps_data module"""
 
+import numpy as np
 import pytest
 
-from forest.bonsai.simulate_gps_data import get_path
+from forest.bonsai.simulate_gps_data import (
```

Comment:
Let's follow this format for multiline imports:
```
from Tkinter import (Tk, Frame, Button, Entry, Canvas, Text,
    LEFT, DISABLED, NORMAL, RIDGE, END)
```
https://www.python.org/dev/peps/pep-0328/